partials/tabs/projects.html

```diff
@@ -26,8 +26,6 @@
                 <br> </div>
               <div ng-show="project.collaborating_projects.length>0" class="project-overview-element clickable card-action"> <span ng-click="search(project)" class="pr-element-detail chip" ng-repeat="project in project.collaborating_projects">{{ project }}</span>
                 <br> </div>
-              <div ng-show="project.status=='crowded'" class="red white-text flag card-action center"> <span class="status-title">HIGH INVOLVEMENT</span>
-                <br> <span class="tiny">If you haven't put work into this, look out for other projects for better chances!</span> </div>
```

Comment:
@alexwaibel yes you are right :) In a recently merged PR, projects metadata was put in individual files of project and projects.js is autogenerated now by jekyll. 
Using jekyll here is a very very recent development and an experiment kind of and prolly thats why Satwik suggested about projects.js. 
yes you need to remove the status:crowded from individual projects :) 👍 .